documentation/ci-fix.md

```diff
@@ -30,6 +30,9 @@ cd specification/contosowidgetmanager
 # Install the dependencies to the local 'node_modules' folder.
 npm install
 
+# Generate the prettier swagger library from type script (the command will return an error which is expected)
+npx tsc
```

Comment:
This has been recently removed by @weshaggard in [this change]. @weshaggard can you comment?

Reply:
This should no longer be required @berndverst are you seeing an particular issue? If so which branch are you trying to run this against? Is that branch updated with main?

Reply:
@weshaggard chances are the branch is not up to date. I probably don't want to be the one updating it - it's my first time having opened a Swagger change PR, so I don't want to run into even more potential problems :)

The branch I'm working with is `release-app-Microsoft.App-2023-08-01-preview`.

Reply:
OK I'll leave that up to you when you want to update the branch or not but we shouldn't add these to the main docs as this step is no longer needed. For your branch if it is needed feel free to do this step. I'm going to close this PR.